Cargo.lock

```diff
@@ -1,5 +1,7 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+version = 3
```

Comment:
Hmm, why it's here now?

Reply:
This is because of https://github.com/rust-lang/cargo/pull/9133. I don't think it is part of beta yet, so you can remove it.

Reply:
Yeah, removed this.